scripts/getDependencies.pl

```diff
@@ -202,7 +202,12 @@
 		url => 'https://dlcdn.apache.org/maven/maven-3/3.9.8/binaries/apache-maven-3.9.8-bin.tar.gz',
 		fname => 'apache-maven-bin.tar.gz',
 		sha1 => '7f15c63c129f036dd5c96b1a591ed8d888f75617'
-	});
+	},
+  dacapo => {
+    url => 'https://download.dacapobench.org/chopin/dacapo-23.11-MR2-chopin-minimal.zip',
+    fname => 'dacapo-23.11-MR2-chopin-minimal.zip', 
```

Comment:
Let's use the file name `dacapo.zip` here. If we need to upgrade the version in the future, we do not need to change the reference code for the dacapo zip name.

Please also fix the indentation. Thanks